docs/sts/aws-sts.md

```diff
@@ -0,0 +1,75 @@
+## Sigstore on OpenShift, with AWS Security Token Service (STS)
+
+[Running pods in OpenShift with AWS IAM Roles for service accounts (IRSA)](https://cloud.redhat.com/blog/running-pods-in-openshift-with-aws-iam-roles-for-service-accounts-aka-irsa)
+
+AWS STS is used to create workload identity tokens for service accounts.
+Sigstore is configured to use the service account OIDC Identity Token to pass to Fulcio to authenticate requests.
+
+[Understanding ROSA with STS](https://docs.openshift.com/rosa/rosa_getting_started/rosa-sts-getting-started-workflow.html)
+
+### Configure Fulcio chart with AWS STS OIDC issuer
+
+In your values file the `scaffold.fulcio` section should include the following:
+
+```yaml
+fulcio:
+  config:
+    contents:
+      OIDCIssuers:
+        # replace
+        ? https://rh-oidc.s3.us-east-1.amazonaws.com/.....
+        : IssuerURL: https://rh-oidc.s3.us-east-1.amazonaws.com/.......
+          ClientID: sigstore
+          Type: kubernetes
+```
+
+### Create a service account and signer deployment
+
+For this you will need to have an `IAM>Role` for your AWS Identity Provider with
+permissions to list S3 buckets. From the AWS Console, choose
+`Roles-> Create Role -> Web Identity`.
+From here you'll need to choose your Identity provider from the dropdown list in your account and Audience.
```

Comment:
Nit: this could be simplified.
```suggestion
Choose your Identity provider from the dropdown list in your account and Audience.
```
I don't understand "in your account and Audience" though.

Reply:
i'll clarify, this is all from the Google Cloud Console, so your Google Cloud account, it doesn't sound quite right, agreed